lib/simple_deploy/notifier.rb

```diff
@@ -36,6 +37,9 @@ def send(message)
           campfire = Notifier::Campfire.new :stack_name  => @stack_name,
                                             :environment => @environment
           campfire.send message
+        when 'slack'
+          slack = Notifier::Slack.new
```

Comment:
I guess it doesn't much matter since this will only be called once per process, but should the notifier really send a message every time?